tests/TestCase/Database/ConnectionTest.php

```diff
@@ -215,21 +215,6 @@ public function testWrongCredentials(): void
         $this->assertInstanceOf('PDOException', $e->getPrevious());
     }
 
-    public function testConnectRetry(): void
-    {
-        $this->skipIf(!ConnectionManager::get('test')->getDriver() instanceof Sqlserver);
-
-        $connection = new Connection(['driver' => 'RetryDriver']);
```

Comment:
This driver triggers a specific failure to ensure the retry-on-failure-code tries again.